public/Census_ACS_5yr/Census_ACS_5yr.py

```diff
@@ -10,22 +10,27 @@ def udf(
     
     from utils import acs_5yr_bounds
 
+    # convert bounds to tile
+    common_utils = fused.load("https://github.com/fusedio/udfs/tree/bb712a5/public/common/").utils
+    zoom = common_utils.estimate_zoom(bounds)
+    tile = common_utils.get_tiles(bounds, zoom=zoom)
+
     # different geometry details per zoom level
-    if bounds.z[0] > 12:
+    if zoom > 12:
         suffix = None
-    elif bounds.z[0] > 10:
+    elif tile.z[0] > 10:
```

Comment:
```suggestion
    elif zoom > 10:
```

and the two cases on the lines below as well

Reply:
And this comment also applies to almost all other UDFs were we currently use `tile.z[0]`. If we already calculated zoom, I think it is simpler to show to use that directly